### PR TITLE
react_router.md - fix file extensions.

### LINKED ISSUE #25971: <React>: <Fix type>
### Describe your suggestion

If I'm right, there is a mistake:

Screenshot 2023-07-26 at 17 51 36

It should be main.jsx, not index.js

### Path

Node / JS

### Lesson Url

https://www.theodinproject.com/lessons/node-path-react-new-react-router

### Checks

- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [X] Would you like to work on this issue?

### (Optional) Discord Name

fgostev

### (Optional) Additional Comments

_No response_

### PR DESCRIPTION
**Pull request by fgostev**
## Because

Issues with ending of files. Switched .js files all to .jsx so it can work with vit.

## This PR

-replaced all .js to .jsx typos
-replaced index.js to main.jsx

## Issue

Closes #25971 

## Pull Request Requirements

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`
-   [x] The `Because` section summarizes the reason for this PR
-   [x] The `This PR` section has a bullet point list describing the changes in this PR
-   [x] If this PR addresses an open issue, it is linked in the `Issue` section
-   [x] If any lesson files are included in this PR, they have been previewed with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure it is formatted correctly
-   [x] If any lesson files are included in this PR, they follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

### CODE REVIEW

**Comment by ManonLef** [comment]
Could you link to the issue this closes please? Thank you :)
Screenshot 2023-07-28 at 12 13 35

**Comment by fgostev** [comment]
> Could you link to the issue this closes please? Thank you :) Screenshot 2023-07-28 at 12 13 35

Done! (_hopefully all correct now, sorry for that_)

File: react/the_react_ecosystem/react_router.md [hunk 1]
```diff
@@ -417,7 +417,9 @@ const Router = () => {
 export default Router;
 ~~~
 
-So, we can simply add this `Router.jsx` component to the `main.jsx` file.
+
+So, we can simply add this `Router.jsx` component to the `Main.jsx` file.
+
```
Comment (ManonLef) [inline react/the_react_ecosystem/react_router.md:422]:
```suggestion
So, we can simply add this `Router.jsx` component to the `Main.jsx` file.
```

removed newly added newlines

**Review by ManonLef — CHANGES_REQUESTED** [review]
Thanks for updating this! I added a suggestion below since the extra lines aren't needed on top of the new lines that were already there.

File: react/the_react_ecosystem/react_router.md (hunk 1 above)
Comment (fgostev) [reply react/the_react_ecosystem/react_router.md]: Added the suggestion - please let me know if everything is all right now =)

**Comment by ManonLef** [comment]
I noticed there is another instance of `main.jsx` in this lesson, would you be willing to go through it and correct that as well? And perhaps check if the other ones are up to date?

**Comment by fgostev** [comment]
Done, went over and submitted a commit!

**Comment by ManonLef** [comment]
Thank you so much!

**Review by ManonLef — APPROVED** [review]

**Comment by fgostev** [comment]
Thanks for reviewing!

**Pull request merged**